source/_components/remember_the_milk.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: "Remember The Milk"
+description: "Instructions on how to use Remember The Milk with Home Assistant."
+date: 2017-10-10 10:10
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
```

Comment:
They have a nice logo. Let's use that to have visual representation of the RTM:

Reply:
I'll ask for permissions. Let's see what happens...

Reply:
@fabaff 
Still no feedback from Remember The Milk about the usage of the logo. Should we just merge it as it is and fix the logo later on once we have the approval?

Reply:
We should have no logo instead then.